d2l-dom.html

```diff
@@ -31,7 +31,7 @@
 				if (node.shadowRoot) {
 					node = node.shadowRoot;
 				}
-				nodes = node.children;
+				nodes = node.children || {};
```

Comment:
Correct, but if we change to use `childNodes` we need to ensure we deal with non-elements, which oddly seems to be handled in two places in the original code.